omnibus/.kitchen.yml

```diff
@@ -11,7 +11,9 @@ driver:
     cpus: 4
     memory: 4096
   synced_folders:
-    - ['../..', '/home/vagrant']
```

Comment:
@jkeiser This matches what we do in all other projects...it appears to work as expected: ```[vagrant@chefdk-centos-72 ~]$ ls -la /home/vagrant/chefdk/
total 193
drwxr-xr-x. 1 vagrant vagrant  1122 Apr 15 14:59 .
drwxr-xr-x. 9 vagrant vagrant  4096 Apr 15 19:05 ..
drwxr-xr-x. 1 vagrant vagrant   238 Apr 15 14:59 acceptance
-rw-r--r--. 1 vagrant vagrant   550 Apr 15 14:59 appveyor.yml
drwxr-xr-x. 1 vagrant vagrant   102 Mar 29  2015 bin
drwxr-xr-x. 1 vagrant vagrant   102 Apr 15 14:59 .bundle
-rw-r--r--. 1 vagrant vagrant 68770 Apr 15 14:59 CHANGELOG.md
-rw-r--r--. 1 vagrant vagrant  2448 Apr 15 14:59 chef-dk.gemspec
drwxr-xr-x. 1 vagrant vagrant   136 Apr 15 14:59 ci
-rw-r--r--. 1 vagrant vagrant  8554 Apr 15 14:59 CONTRIBUTING.md
-rw-r--r--. 1 vagrant vagrant   374 Nov 21 20:35 DOC_CHANGES.md
-rw-r--r--. 1 vagrant vagrant  3108 Apr 15 14:59 Gemfile
-rw-r--r--. 1 vagrant vagrant 17030 Apr 15 14:59 Gemfile.lock
-rw-r--r--. 1 vagrant vagrant  1112 Apr 15 14:59 Gemfile.windows
-rw-r--r--. 1 vagrant vagrant 22361 Apr 15 14:59 Gemfile.windows.lock
drwxr-xr-x. 1 vagrant vagrant   544 Apr 15 19:14 .git
-rw-r--r--. 1 vagrant vagrant   484 Apr 15 14:59 .gitignore
-rw-r--r--. 1 vagrant vagrant   319 Mar 29  2015 Guardfile
drwxr-xr-x. 1 vagrant vagrant   170 Mar 29  2015 lib
-rw-r--r--. 1 vagrant vagrant 10850 Mar 29  2015 LICENSE
drwxr-xr-x. 1 vagrant vagrant   544 Apr 15 18:52 omnibus
-rw-r--r--. 1 vagrant vagrant   423 Apr 15 14:59 omnibus_overrides.rb
-rw-r--r--. 1 vagrant vagrant   226 Apr 15 14:59 POLICYFILE_README.md
-rw-r--r--. 1 vagrant vagrant   140 Sep  3  2015 PROVISION_README.md
-rw-r--r--. 1 vagrant vagrant   826 Apr 15 14:59 Rakefile
-rw-r--r--. 1 vagrant vagrant 23561 Apr 15 14:59 README.md
-rw-r--r--. 1 vagrant vagrant  1535 Apr 15 14:59 RELEASE_NOTES.md
-rw-r--r--. 1 vagrant vagrant    25 Mar 29  2015 .rspec
drwxr-xr-x. 1 vagrant vagrant   204 Apr 15 14:59 spec
drwxr-xr-x. 1 vagrant vagrant   306 Apr 15 14:59 tasks
-rw-r--r--. 1 vagrant vagrant  1410 Apr 15 14:59 .travis.yml
-rw-r--r--. 1 vagrant vagrant  3170 Apr 15 14:59 version_policy.rb
-rw-r--r--. 1 vagrant vagrant   404 Nov 21 20:35 warning.txt```